users/models.py

```diff
@@ -274,3 +275,41 @@ def create_last_seen_for_user(sender, instance, created, raw, **kwargs):
 
     if created:
         LastSeen.objects.create(user=instance, at=timezone.now())
+
+
+class InvitationToken(models.Model):
+    """
+    An invitation token to create a user account.
+    """
+
+    user = models.ForeignKey(
+        settings.AUTH_USER_MODEL,
+        on_delete=models.CASCADE,
+        related_name="invitations",
+        help_text="The user who was invited to create an account.",
```

Comment:
I'm not super familiar with this model so this help message is a bit confusing: if the User instance already exists so we can FK to it, how's the user then invited to create the account? Is it perhaps a way to claim accounts?

Reply:
Yes, we have to pre-create the User objects for the github users we find as authors/maintainers/etc of the git repos and then have a process for them to claim the account later.